project/Build.scala

```diff
@@ -29,7 +29,7 @@ object Build extends Build {
   val lib = project("common-lib")
     .libraryDependencies(loggingDeps ++ awsDeps ++ elasticsearchDeps ++
       playDeps ++ playWsDeps ++ scalazDeps ++ commonsIODeps ++ akkaAgentDeps ++
-      pandaDeps ++ imagingDeps)
+      pandaDeps ++ imagingDeps ++ reactiveXDeps)
```

Comment:
Created an object in common lib that provides a single threaded scheduler so this seems natural.